model/orion/dffml_model_orion/orion_model.py

```diff
@@ -0,0 +1,231 @@
+import pathlib
+from typing import Type
+
+import pandas as pd
+from orion import Orion
+from orion.data import load_signal
+
+from dffml import (
+    config,
+    field,
+    entrypoint,
+    SimpleModel,
+    ModelNotTrained,
+    Accuracy,
+    Feature,
+    Features,
+    SourcesContext,
+    Record,
+)
+
+
+# Hyperparameters are passed in as tuples
+def tup_to_dict(tup):
+    tup = (tup,)
+    dic = {}
+    for param in tup:
+        dic[param[0]] = param[1]
+    return dic
+
+
+@config
+class OrionModelConfig:
+    data: str = field(
+        "Time sequence data (timestamps and corresponding values) to be put into the orion model as a csv file"
+    )
+    predict: str = field(
+        "The start and end values of your ground truth as a csv file",
+        default="",
+    )
+    accuracy: str = field("The ground truth ", default="")
```

Comment:
Remove these and replace with taking data from `sources`, ensure callers via CLI or Python API are passing in sources pointing to the files we are currently giving via strings to the config.